### ISSUE TITLE
Git problem

### ISSUE THREAD

**Issue by d954mas** [defold/editor2-issues#2342]
I work with my teammate. Then happened strange thing. I can't commit because editor not respone when i try to push. I make some commits by command line.   Editor can't push but can pull. For now when i try to pull, i get error getting changes.

### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2018-11-28T10:40:18.269998</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>496c3b0afba64917c0993ed33be59c081d2ad99a</td></tr>
<tr><td>Defold engine sha</td><td>6230c40dfa051a311aa4005c19097574e062374d</td></tr>
<tr><td>Defold version</td><td>1.2.142</td></tr>
<tr><td>GPU</td><td>Intel(R) HD Graphics 4600</td></tr>
<tr><td>GPU Driver</td><td>4.2.0 - Build 10.18.10.4425</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 7</td></tr>
<tr><td>OS version</td><td>6.1</td></tr>
<table>

**Comment by d954mas** [comment]
project https://yadi.sk/d/5fgNhDTUTW08pg

**Comment by Jerakin** [comment]
The repository is mid merge with a conflict. Resolve the merge and commit it and you should be able to progress.

**Comment by d954mas** [comment]
ok. What about editor. If i not wrong, it can merge?

**Comment by Jerakin** [comment]
Yes, the editor (or rather git) can merge many changes by itself. But if you and your collaboration partner have worked on the same file on the same line - git will have problems auto merging.

**Comment by d954mas** [comment]
yes i know it. I don;t tell about auto merge. Maybe i am wrong. But i remeber that if i have conflict, editor tell me that i have conflicts, and i can resolve them

**Comment by d954mas** [comment]
I resolve conflict. It fixed error getting changes. But i can't push. The editor is not responce. Also update project.

**Comment by Jerakin** [comment]
What happens if you do it from the command line?

**Comment by d954mas** [comment]
it worked, from command line

**Comment by d954mas** [comment]
I restart editor. Now it worked. So looks like it can't push while have conflicts. Or something like that. 
Thanks @Jerakin

**Comment by erikangelin-king** [comment]
Note: we should detect externally created git conflicts and inform about them in the sync dialog.
